scripts/partition_manager.py

```diff
@@ -796,6 +812,28 @@ def test():
     offset = get_required_offset(align={'end': 800}, start=0, size=1000, move_up=False)
     assert offset == 600
 
+    failed = False
+    try:
+        offset = get_required_offset(align={'end': ["CONFIG_VAR"]}, start=0, size=1000, move_up=False)
+    except TypeError:
+        assert True
+    else:
+        assert False
+
+    try:
+        offset = get_required_offset(align={'start': ["CONFIG_VAR"]}, start=0, size=1000, move_up=False)
+    except TypeError:
+        assert True
+    else:
```

Comment:
AFAICS this is python 3.8, and zephyr only support 3.6. I think you need to do the good old variable setting and checking

Reply:
removed